.github/workflows/ci_full_downstream_build.yml

```diff
@@ -115,138 +102,37 @@ jobs:
             echo $MERGE_WARNING_MESSAGE
             git merge ${{ steps.buildInfo.outputs.user }}/${{ steps.buildInfo.outputs.ref }}
           fi
+          
+      - name: "kogito-tooling :: Setup Env"
+        uses: ./kogito-tooling/.github/actions/setup-env
+        with:
+          path: ${{ github.workspace }}/kogito-tooling
 
       - name: "kogito-tooling :: Build"
         env:
+          WEBPACK__minimize: "false"
+          WEBPACK__tsLoaderTranspileOnly: "false"
+          KOGITO_TOOLING_BUILD_lint: "true"
+          KOGITO_TOOLING_BUILD_test: "true"
+          KOGITO_TOOLING_BUILD_testIT: "true"
+          KOGITO_TOOLING_BUILD_docker: "true"
+          KOGITO_TOOLING_BUILD_examples: "true"
           DISPLAY: ":99.0"
+          START_SERVER_AND_TEST_INSECURE: "true"
         run: |
-          cd ${{ github.workspace }}/kogito-tooling
-          yarn bootstrap && yarn build:prod
+          lerna run build:prod --stream --concurrency 1
```

Comment:
so no `yarn bootstrap` before we begin building? Is that expected?

Reply:
It's part of the `setup-env` action on `kogito-tooling`.

Reply:
How is that action run on `kogito-editors-java` ? if it is a action of `kogito-tooling` am I missing something?

Reply:
But imho this part will be missing `cd ${{ github.workspace }}/kogito-tooling` the error in #114 suggests that